stl/inc/flat_set

```diff
@@ -206,16 +203,18 @@ public:
 
     // capacity
     _NODISCARD_EMPTY_MEMBER bool empty() const noexcept {
-        return _Get_cont().empty();
+        return _Mycont.empty();
     }
     _NODISCARD size_type size() const noexcept {
-        return _Get_cont().size();
+        return _Mycont.size();
     }
     _NODISCARD size_type max_size() const noexcept {
-        return _Get_cont().max_size();
+        return _Mycont.max_size();
     }
 
     // modifiers
+    // TRANSITION, the "insert" and "erase" methods may not be able to restore the invariant, if the underlying
+    // container is unable to provide strong guarantee for "erase" and "insert" methods.

```

Comment:
These are related to #4059. Aside from exception guarantee, I think the standard is also not clear enough about how should the invariant be restored.
>If any member function in [[flat.set.defn]](https://eel.is/c++draft/flat.set#defn) exits via an exception, the invariant is restored[.](https://eel.is/c++draft/flat.set#overview-6.sentence-1)
[Note [2](https://eel.is/c++draft/flat.set#overview-note-2): This can result in the flat_set's being emptied[.](https://eel.is/c++draft/flat.set#overview-6.sentence-2) — end note]

Should `flat_meow` be unchanged, or should `flat_meow` be emptied? I think the standard should really give some specifications. At least for erasion and single-element insertion, strong-guarantee is possible if the underlying container provide strong-guarantee too.